### PR TITLE
Add Jubaan's bio, correct indentation.

### PR DESCRIPTION
**Pull request by jubaan**
This will add my (jubaan) bio to the Exercism Ruby Mentors.

### CODE REVIEW

**Comment by SleeplessByte** [comment]
Hi there. Next time you can update the _branch_ you previously pushed to, in order to _update a PR_. If you need more information regarding this subject, let us know, and we'll help you find helpful documentation.

File: mentors/ruby/j.json [hunk 1]
```diff
@@ -62,5 +62,13 @@
...
     "bio": "After years of light web development in the financial industry, I changed course to follow my passion for software development. I love working with Ruby and Javascript and am currently building my knowledge around Elixir, Java, React, and Angular."
-  }  
+  },
+  {
+    "github_username": "jubaan",
+    "name": "Julio Añoveros",
+    "link_text": null,
+    "link_url": null,
+    "avatar_url": null,
+    "bio": "Full-stack Web Developer, Ruby entusiast and avid learner. My phylosophy: 'Teaching as a way of learning'."
```
Comment (SleeplessByte) [inline mentors/ruby/j.json:72]:
```suggestion
    "bio": "Full-stack Web Developer, Ruby enthusiast and avid learner. My philosophy: 'Teaching as a way of learning'."
```

**Review by SleeplessByte — DISMISSED** [review]
See inline

**Review by rpottsoh — APPROVED** [review]

**Comment by SleeplessByte** [comment]
Welcome to Ruby!

**Pull request merged**